src/store/middlewares/account.js

```diff
@@ -209,6 +211,7 @@ const accountMiddleware = store => next => (action) => {
       votePlaced(store, action);
       break;
     case actionTypes.accountLoggedOut:
+      store.dispatch(settingsUpdated({ token: { active: tokenMap.LSK.key } }));
```

Comment:
It would also be nice to add a comment here explaining why we switch the token when logging out (Also add the issue no may be). Looking at this, it is not very obvious why we do so.